packages/js/product-editor/src/components/header/publish-button/publish-button.tsx

```diff
@@ -24,14 +25,17 @@ export function PublishButton(
 		'status'
 	);
 
-	const isCreating = productStatus === 'auto-draft';
+
+	const isCreating = status === 'auto-draft';
```

Comment:
Where `status` variable is coming from?

Reply:
Accidentally left over from the previous iteration before I moved the tracking utils out.  So weird my compiler didn't complain.  Fixed!